### ISSUE TITLE
Exception on tape.gradient()

### ISSUE THREAD

**Issue by aschoenebeck** [SciSharp/TensorFlow.NET#843]
Using huber loss keras.losses.Huber().

```
    <PackageReference Include="SciSharp.TensorFlow.Redist" Version="2.6.0" />
    <PackageReference Include="TensorFlow.Keras" Version="0.6.2" />
    <PackageReference Include="TensorFlow.NET" Version="0.60.2" />
```

```
System.Collections.Generic.KeyNotFoundException
  HResult=0x80131577
  Message=The given key 'SelectV2' was not present in the dictionary.
  Source=System.Private.CoreLib
  StackTrace:
   at System.ThrowHelper.ThrowKeyNotFoundException[T](T key)
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Tensorflow.Eager.EagerRunner.<>c__DisplayClass5_0.<GetGradientFunction>b__0(Tensor[] output_grads, Int64[] unneeded_gradients)
   at Tensorflow.Gradients.Tape.CallBackwardFunction(BackwardFunction backward_function, List`1 unneeded_gradients, List`1 output_gradients)
   at Tensorflow.Gradients.Tape.ComputeGradient(Int64[] target_tensor_ids, Int64[] source_tensor_ids, UnorderedMap`2 sources_that_are_targets, Tensor[] output_gradients)
   at Tensorflow.Gradients.GradientTape.gradient(Tensor target, IEnumerable`1 sources)
   at ...
```

**Comment by Oceania2018** [comment]
@aschoenebeck Can you try v0.60.5?
